src/porepy/models/constitutive_laws.py

```diff
@@ -676,35 +676,58 @@ class CubicLawPermeability(ConstantPermeability):
 
     """
 
-    def permeability(self, subdomains: list[pp.Grid]) -> np.ndarray:
+    def permeability(self, subdomains: list[pp.Grid]) -> pp.ad.Operator:
         """Permeability [m^2].
 
```

Comment:
Please leave a comment how the permeability is defined (separately for matrix and fractures).